Go 1.16 Deprecates IOUtil; lint fails build

### What happened?

Building from master using go 1.19.5 (released October 2022) fails lint with the error in the "reproduction" section.

### What did you expect to happen?

Build from master should succeed.

### How can we reproduce this?

```sh
~/IdeaProjects/pluto/ git:(master) $ make
golangci-lint run
cmd/root.go:34:2: SA1019: "io/ioutil" has been deprecated since Go 1.16: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details. (staticcheck)
	"io/ioutil"
	^
pkg/finder/finder.go:33:2: SA1019: "io/ioutil" has been deprecated since Go 1.16: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details. (staticcheck)
	"io/ioutil"
	^
make: *** [lint] Error 1
~/IdeaProjects/pluto/ git:(master) $ go version  
go version go1.19.5 darwin/arm64
```

### Version

1.19.5

### Search

- [X] I did search for other open and closed issues before opening this.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Additional context

will be submitting a potential fix for this shortly; it seems an easy fix

--- a/cmd/root.go
+++ b/cmd/root.go
@@ -31,7 +31,7 @@ package cmd
 import (
 	"flag"
 	"fmt"
-	"io/ioutil"
+	"io"
 	"os"
 	"strings"
 
@@ -195,7 +195,7 @@ var rootCmd = &cobra.Command{
 		var deprecatedVersionList []api.Version
 		if additionalVersionsFile != "" {
 			klog.V(2).Infof("looking for versions file: %s", additionalVersionsFile)
-			data, err := ioutil.ReadFile(additionalVersionsFile)
+			data, err := os.ReadFile(additionalVersionsFile)
 			if err != nil {
 				return err
 			}
@@ -349,7 +349,7 @@ var detectCmd = &cobra.Command{
 
 		if args[0] == "-" {
 			//stdin
-			fileData, err := ioutil.ReadAll(os.Stdin)
+			fileData, err := io.ReadAll(os.Stdin)
 			if err != nil {
 				fmt.Println("Error reading stdin:", err)
 				os.Exit(1)

--- a/pkg/finder/finder.go
+++ b/pkg/finder/finder.go
@@ -30,7 +30,6 @@ package finder
 
 import (
 	"fmt"
-	"io/ioutil"
 	"os"
 	"path/filepath"
 
@@ -122,7 +121,7 @@ func (dir *Dir) scanFiles() error {
 // it is an api-versioned Kubernetes object.
 // Returns the File object if it is.
 func (dir *Dir) CheckForAPIVersion(file string) ([]*api.Output, error) {
-	data, err := ioutil.ReadFile(file)
+	data, err := os.ReadFile(file)
 	if err != nil {
 		return nil, err
 	}

